### DISCUSSION TITLE
GPIO Pins iobroker Docker

### DISCUSSION THREAD

**Discussion by chesterb87** [buanet/ioBroker.docker#263] [Questions & Answers / Fragen & Antworten] [answered]
Hallo,

ist es möglich, die GPIO Pins eines Raspberry PIs aus dem iobroker Docker Container anzusprechen / auszulesen?

Ich erhalte in dem RPi Adapter immer folgende Meldung:

"Cannot setup port 17 as input: Error: EACCES: permission denied, open '/sys/class/gpio/export'"

folgendes habe ich schon probiert nach suchen/googlen:

/sys durchgereicht in den Container
Container im priviliged mode gestartet
/dev/mem & /dev/gpiomem in den Container durchgereicht

Leider immer wieder der selbe Fehler. Hat da jemand eine Idee? 

Ich hab auch in dem Verzeichnis mal die Berechtigung für ANY testweise angepasst. Wenn ich dann den Wert über den Datenpunkt schreibe, erhalte ich folgende Meldung:

Error: ENOENT: no such file or directory, open '/sys/class/gpio/gpio17/value'

Es muss auch nicht zwingend über den Adapter gehen. Ich muss nur irgendwie über ein blockly script den GPIO 17 schalten & auslesen können 🙂

Danke & Gruß

**Comment by buanet** [comment, marked as answer]
Schau mal #112. Wurde dort diskutiert. Leider hat sich niemand gefunden der mal eine Lösung erarbeitet hat.

MfG,
André
